.github/workflows/lock-release.yml

```diff
@@ -24,36 +27,52 @@ jobs:
           owner: primer
           repositories: react
           private-key: ${{ secrets.PRIMER_APP_PRIVATE_KEY_SHARED }}
-      - name: Toggle Rule > Release Conductor
+      - name: Toggle rulesets
         run: |
-          enforcement=$([ "${{ github.event.inputs.action }}" == "lock" ] && echo "active" || echo "disabled")
+          # Allow react-release-conductor to bypass merge queue
+          gh api \
+            --method PUT \
+            -H "Accept: application/vnd.github+json" \
+            -H "X-GitHub-Api-Version: 2022-11-28" \
+            /repos/primer/react/rulesets/4089335 \
+            -F "bypass_actors[][actor_id]=12276524" \
+            -f "bypass_actors[][actor_type]=Team" \
+            -f "bypass_actors[][bypass_mode]=always"
```

Comment:
This lets react-release-conductor bypass the merge queue when lock is in effect. We have to do this because if we leave the merge queue on the checkbox won't showup to bypass unless the conductor can also bypass the queue.